script.js

```diff
@@ -0,0 +1,193 @@
+// Menu Popup
+const menu = document.querySelector('#menu');
+const pop = document.querySelector('.pop-menu');
+
+function poping() {
+  if (pop.style.display === 'block') {
+    menu.src = './images/Enabled.svg';
+    pop.style.display = 'none';
+    menu.style.margin = '3%';
+  } else {
+    menu.src = './images/X-Icon.svg';
+    pop.style.display = 'block';
+    menu.style.margin = '6%';
+  }
+}
+
+menu.addEventListener('click', poping);
+
+const menus = document.querySelectorAll('.pop-menu a');
+for (let i = 0; i < menus.length; i += 1) {
+  menus[i].addEventListener('click', () => {
+    poping();
+  });
+}
+
+// Projects Generation and Popup
+const projects = [
+  {
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    mobileImage: './images/Snapshoot Portfolio.svg',
+    desktopImage: './images/Snapshoot Portfolio 1.svg',
+    technologies: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'CANOPY',
+    year: '2015',
+    stack: 'Back End Dev',
+  },
+  {
+    name: 'Post Stories',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    mobileImage: './images/Snapshoot Portfolio (1).svg',
+    desktopImage: './images/Snapshoot Portfolio 2.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'FACEBOOK',
+    year: '2015',
+    stack: 'Full Stack Dev',
+  },
+  {
+    name: 'Facebook 360',
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    mobileImage: './images/Snapshoot Portfolio (2).svg',
+    desktopImage: './images/Snapshoot Portfolio 3.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'FACEBOOK',
+    year: '2015',
+    stack: 'Full Stack Dev',
+  },
+  {
+    name: 'Uber Navigation',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    mobileImage: './images/Snapshoot Portfolio (3).svg',
+    desktopImage: './images/Snapshoot Portfolio 4.svg',
+    technologies: [
+      'HTML',
+      'Ruby on Rails',
+      'CSS',
+      'JavaScript',
+    ],
+    liveVersionLink: '',
+    SourceLink: '',
+    company: 'Uber',
+    year: '2018',
+    stack: 'Lead Developer',
+  },
+];
```

Comment:
- [ ] It is required to stick to the figma design as close as possible. This includes margin, padding typography among others. Your design looks a bit different from the figma design. Kindly look into below two screenshots and walk your design alongside the figma desing both on the desktop and the mobile design.

### Your Design (Desktop)
![your-design](https://github.com/SnakyMz/Portfolio/assets/26597432/dc820cf9-7017-46cb-9427-784728ea5524)

### Figma Design (Desktop)
![figma-design-desktop](https://github.com/SnakyMz/Portfolio/assets/26597432/0c33514d-3079-4bda-be35-b44797b01e28)



### Your Design (Mobile)
![your-design-mobile](https://github.com/SnakyMz/Portfolio/assets/26597432/f12a4d84-188d-4a74-8595-1b7b3a61ea8e)

### Figma Design (Mobile)
![figma-design-mobile](https://github.com/SnakyMz/Portfolio/assets/26597432/76b34d51-f07d-4392-a41f-477df90f63e5)

Reply:
If you screen width in the mobile resolution. My design is similar to the figma. The picture you have provided for my mobile device is taken on a very wide resolution. Please review with modesty, don't be lacy about. As for the desktop version, I admit it's not similar to figma and I am making changes to it. And for project description part its more functional as it display details of the selected project rather than just having lorem ipsum as text.